aeron-client/src/main/c/concurrent/aeron_logbuffer_descriptor.h

```diff
@@ -71,6 +71,7 @@ typedef struct aeron_logbuffer_metadata_stct
     uint8_t signal_eos;
     uint8_t spies_simulate_connection;
     uint8_t tether;
+    uint8_t pad_end[32]; // Padding to align the structure size to 512 bytes (9x AERON_CACHE_LINE_LENGTH)
```

Comment:
Why add trailing padding? The entire metadata section is 4KB and is cache line aligned. We are going to read only the fields we need, i.e. consume up to 480 bytes.

Reply:
The original code also had padding at the end of the record:

```
uint8_t pad3[(AERON_CACHE_LINE_LENGTH) - (7 * sizeof(int32_t))];
```

My guess is that it was done to prevent false sharing.

Reply:
I had another look. I think I understand what is happening.

The padding is needed to get the header at the right position. It isn't needed to prevent false sharing.

Since the empty space (the pad3) is now filled with the extra fields, the padding isn't relevant any longer.

Reply:
Yes, so no extra padding is needed. The definition on the C side was correct after the https://github.com/real-logic/aeron/commit/9eb689f6ea6a4e935bda5ce7732413adfcc808c9 commit.